src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Emitter.cs

```diff
@@ -254,11 +254,8 @@ private void GenCases(LoggerMethod lm, string nestedIndentation)
                 {
                     // this is related to https://github.com/serilog/serilog-extensions-logging/issues/197
                     string name = p.CodeName;
-                    if (lm.TemplateMap.ContainsKey(name))
-                    {
-                        // take the letter casing from the template
-                        name = lm.TemplateMap[name];
-                    }
+                    // take the letter casing from the template
+                    lm.TemplateMap.TryGetValue(name, out name);
```

Comment:
if the name doesn't exist, this will result in nulling it out rather than keeping CodeName, no?

Reply:
Right, thanks